src/cargo-fmt/main.rs

```diff
@@ -345,6 +346,11 @@ fn run_rustfmt(
             h
         });
 
+    let s = "2018".to_string();
```

Comment:
I think it'd be better to use the default of `Edition`.

Reply:
I'm not sure I understand.  `Edition` doesn't implement anything like Display to get a string.  Also, this bin currently doesn't import anything from rustfmt, so that would be a distinct change.

Reply:
Indeed. However I don't really like scattering defaults all over, especially here where it'd be to get around a piece of code. With a bit of refactoring, it'll be nicer:

- we are already executing `rustfmt` without the need for edition in order to get the version:
     https://github.com/rust-lang/rustfmt/blob/466ddc9aba88dd33e28226642223c5285c47336a/src/cargo-fmt/main.rs#L87-L89 
- we could refactor a bit `get_version` so that it can execute other option beside `version`
    https://github.com/rust-lang/rustfmt/blob/466ddc9aba88dd33e28226642223c5285c47336a/src/cargo-fmt/main.rs#L120 
- this way the `if` block wouldn't be needed since any of the options (`--help`, `-V` ...) would be handled by the refactored `get_version` method https://github.com/rust-lang/rustfmt/blob/466ddc9aba88dd33e28226642223c5285c47336a/src/cargo-fmt/main.rs#L145-L149

How does that sound ?